cmake/alpakaCommon.cmake

```diff
@@ -55,6 +55,7 @@ option(ALPAKA_ACC_CPU_B_TBB_T_SEQ_ENABLE "Enable the TBB CPU grid block back-end
 option(ALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLE "Enable the OpenMP 2.0 CPU grid block back-end" OFF)
 option(ALPAKA_ACC_CPU_B_SEQ_T_OMP2_ENABLE "Enable the OpenMP 2.0 CPU block thread back-end" OFF)
 option(ALPAKA_ACC_ANY_BT_OMP5_ENABLE "Enable the OpenMP 5.0 CPU block and block thread back-end" OFF)
+option(ALPAKA_ACC_ANY_BT_OACC_ENABLE "Enable the OpenACC block and block thread back-end" OFF)
```

Comment:
Consistency with the OpenMP backends. I think these name allude to an early idea that one could combine backends for thread and block level, so backends always say, that they handle both block and thread-level, not allowing combination with others. Changing this would require a separate discussion and should be done for all backends.

Reply:
Ok!